spec/ui/home_page_spec.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+describe Pages::Home do
+  context 'Dfe Homepage' do
+    before(:each) do
+      @home_page = Pages::Home.new
+    end
```

Comment:
OK - I maybe getting into a big debate here but ....

My belief is that using `let` is preferable to using instance variables in rspec tests. So this would become:
```ruby
let(:home_page) { Pages::Home.new }
```
And the specs would reference `home_page` as needed. So for example:
```ruby
    it 'Should be displayed when the user navigates to the hame page and clicks the header logo' do
      home_page.load
      home_page.header.logo.click

      expect(home_page).to be_displayed
    end
```

Reply:
For now, the on-demand approach to obtaining an instance for the class is working, the API for the tests will change, and as such this may also require a change, that will unveil itself later I'm sure.